### DISCUSSION TITLE
New Installation. 2 questions/issues

### DISCUSSION THREAD

**Discussion by bhantegv** [mynaparrot/plugNmeet-server#417] [Q&A] [answered]
Hello.

1. Android - 
Screen share cannot be seen unless I change to 'Desktop' view. Is there a way to fix/change this? This would make recording interviews user friendly.

2. We integrated with WordPress. Can I use plugnmeet without WordPress? In other words as stand alone instance? For example: use  meet.website.com as public platform? Where people can use/login without going through an app?

Overall seems to be working OK. Is it possible to connect with a programmer to develop further?

Thank you kindly.

**Comment by jibon57** [comment, marked as answer]
1. Android - ===> From my end I can't find any problem. Screen share is loading just fine. But from mobile device you won't be share screen. Not all browser support it. 

2. Can I use plugnmeet without WordPress? ==> Yes, of course. Follow the API for implementation. We don't have any plan to create separate platform instated of we've planned to create plugins targeting different existing platforms. 

Is it possible to connect with a programmer to develop further?
> This is an open source project. Contributors are always welcome.

**Reply by bhantegv** [reply]
Yes on my Android unless you change to desktop view screenshare is not available.

So you are saying that plugNmeet can only be used with apps. Not alone? That's a shame. 

I wonder what could be done to make that happen? Would be a great option to have. 

Thank you.
